instructure/canvas-ios
# Preserve PDF annotations even when cache is reset

refs: [MBL-18069](https://instructure.atlassian.net/browse/MBL-18069)
affects: Student
release note: PDF annotations is preserved even when cache is reset

## Test Plan
* Login to Student app
* Open a PDF file and make some annotations to it, then go back.
* Close the app entirely.
* Go to the App's page on Settings app.
* Turn on "Reset cache on next launch" toggle.
* Launch the app, then login back to the same user. 
* Open the PDF file that was annotated in step 2.
* You should see the previously saved annotations on that file.

## Video Record

Recording covering PDF annotation persistence scenarios, which are: 
- On logout
- On cache reset.

https://github.com/user-attachments/assets/6c5e7355-bbff-4950-8fbc-982cb9d5f0fa

## Checklist



[MBL-18069]: https://instructure.atlassian.net/browse/MBL-18069?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
